tools/plot_chart.py

```diff
@@ -0,0 +1,135 @@
+import matplotlib.pyplot as plt
+import json
+
+
+def plot_bar_chart(data_points, dest_file):
```

Comment:
this is not some random bar chart, can you give it a very specific name? like, model memory footprint chart or smth like that